.gitignore

```diff
@@ -0,0 +1,20 @@
+
+# Environment variables
+*.env
+
+# Virtual environments
+venv/
+.venv/
+
+__pycache__
+.pytest_cache
+.coverage
+.DS_Store
```

Comment:
What is this?

Reply:
`.DS_Store`, short for Desktop Services Store, is a hidden file created by MacOS to store custom attributes of a folder, such as the positions of icons or the choice of a background image. Whenever you open a folder using Finder, MacOS generates this file to remember your view settings for that particular folder.